File: dbflow/src/androidTest/java/com/raizlabs/android/dbflow/test/modelcontainer/PassType.java
package com.raizlabs.android.dbflow.test.modelcontainer;

import com.raizlabs.android.dbflow.annotation.Column;
import com.raizlabs.android.dbflow.annotation.PrimaryKey;
import com.raizlabs.android.dbflow.annotation.Table;
import com.raizlabs.android.dbflow.structure.BaseModel;
import com.raizlabs.android.dbflow.test.TestDatabase;

/**
 * Description:
 */
@Table(database = TestDatabase.class)
public class PassType extends BaseModel {

    @Column
    @PrimaryKey
    long pid;
}

File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/test/modelcontainer/PassType.kt
package com.raizlabs.android.dbflow.test.modelcontainer

import com.raizlabs.android.dbflow.annotation.Column
import com.raizlabs.android.dbflow.annotation.PrimaryKey
import com.raizlabs.android.dbflow.annotation.Table
import com.raizlabs.android.dbflow.structure.BaseModel
import com.raizlabs.android.dbflow.test.TestDatabase

/**
 * Description:
 */
@Table(database = TestDatabase::class)
class PassType : BaseModel() {

    @Column
    @PrimaryKey
    var pid: Long = 0
}
